resources/public/api/conf/2.0/examples/uk_property_non_fhl_period_summary_create/response.json

```
@@ -0,0 +1,15 @@
+{
+  "periodId": "2017-04-06_2017-07-04",
```

The `periodId` should match the value of the `hrefs` below:
```suggestion
  "periodId": "2019-03-11_2020-04-23",
```